pulseapi/profiles/serializers.py

```diff
@@ -57,6 +57,11 @@ def __init__(self, instance=None, *args, **kwargs):
             # end up in the actual serialized profile data.
             self.fields.pop('enable_extended_information')
 
+    profile_id = serializers.SerializerMethodField()
```

Comment:
probably don't need a `methodfield` for this, you can just do:
```
profile_id = serializers.ReadOnlyField(source='id')
```

Reply:
oh nice, that is even smaller.